app/api/v1/dependencies.py

```
@@ -0,0 +1,47 @@
+from fastapi import Depends
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+
+from app.schemas.user import UserData
+from app.utils.enums import UserRoles
+from app.utils.jwt import decode_token
+
+security = HTTPBearer()
+
+
+async def get_current_user(
+	credentials: HTTPAuthorizationCredentials = Depends(security),
```

Ruff (0.15.18)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)